css_stylesheet.css

```diff
@@ -0,0 +1,257 @@
+	width: 100%;
+}
+
+.portfolioButtons {
+	box-sizing:border-box;
+	display: inline-block;
+	margin-top: 0.4em;
+	padding: 7rem 0rem;
+	width: 25%;
+	background-color: #5C7CFA;
+	border: solid 0.01em #343A40;
+	font-size: 1.5em;
+	font-family: "Courier New", serif;
+	text-align: center;
+	color: #FFFFFF;
+	text-decoration: none;
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: center;
+}
+
+#button1 {
+	background-image: url(https://images.unsplash.com/photo-1583193628511-26907c67dae9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button2 {
+	background-image: url(https://images.unsplash.com/photo-1583167611259-ccfff445b4c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button3 {
+	background-image: url(https://images.unsplash.com/photo-1583228903945-0c003a1b29be?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button4 {
+	background-image: url(https://images.unsplash.com/photo-1558981806-ec527fa84c39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button5 {
+	background-image: url(https://images.unsplash.com/photo-1583183398510-07243101a1fa?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button6 {
+	background-image: url(https://images.unsplash.com/photo-1583129794521-437167ce03a4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button7 {
+	background-image: url(https://images.unsplash.com/photo-1583224874248-9d0981e894d3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+#button8 {
+	background-image: url(https://images.unsplash.com/photo-1583221742001-9ad88bf233ff?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60);
+}
+
+.portfolioButtons:hover, 
+.portfolioButtons:active {
+	cursor: pointer;
+	box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24), 0 17px 50px 0 rgba(0,0,0,0.19);
+}
+
+.pressHereAnchor {
+	font-weight: bold;
+	font-size: 5em;
+	text-decoration: none;
+	color: #DCDCDC;
+	width: 100%;
+}
+
+.pressHereAnchor:hover, 
+.pressHereAnchor:active {
+ 	color: #343A40;
+	cursor: pointer;
+}
+
+.anchorBar {
+	padding: 1.8em;
+}
+
+footer {
+	margin-top: 1em;
+	padding: 1em;
+}
+
+.textAdjust {
+	text-align: center;
+}
+
+#astronaut {
+	color: #26547C;
+}
+
+#code {
+	color: #EF476F;
+}
+
+#briefcase {
+	color: #FFD166;
+}
+
+#biking {
+	color: #06D6A0;
+}
+
+#palette {
+	color: #BF3ED6;
+}
+
+#gamepad {
+	color: #343A40;
+}
+
+@media screen and (min-width: 1024px) {
+	
+	.portfolioButtons {
+		width: 24.3%;
+	}		
+	
+}
+
+@media screen and (max-width: 1024px) and (min-width:770px) {
+	
+	.portfolioButtons {
+		width: 24%;
+	}		
+	
+	.paragraphs {
+		display:block
+	}
+	
+}
+
+@media screen and (max-width: 770px) {
+
+	.portfolioButtons {
+		width: auto;
+	}		
+		
+	.paragraphs {
+		display:block
+	}
+	
+	.portfolioButtons {
+		width: 100%;
+		flex-direction: column;
```

Comment:
This isn't using flex so the flex-direction can be removed

Reply:
stray code from previous iteration.